internal/ntconstants/constants.go

```diff
@@ -0,0 +1,84 @@
+package ntconstants
+
+import (
+	"syscall"
```

Comment:
Package `syscall` is deprecated. For new code prefer `golang.org/x/sys/windows`.